### PR TITLE
fix: DomainSystem resolved & owner

### PR DESCRIPTION
**Pull request by ZhongFuze**
fix: DomainSystem resolved & owner

### CODE REVIEW

File: src/controller/graphql/resolve.rs [hunk 1]
```diff
@@ -130,54 +80,56 @@ impl ResolveQuery {
...
-        );
-        match Resolve::find_by_ens_name(&pool, &name).await? {
-            None => {
-                fetch_all(target).await?;
-                Resolve::find_by_ens_name(&pool, &name).await
-            }
-            Some(resolve) => {
-                if resolve.is_outdated() {
-                    tokio::spawn(fetch_all(target));
+        if domain_system == DomainNameSystem::ENS {
```
Comment (nykma) [inline src/controller/graphql/resolve.rs:98]: use `match` instead.

**Review by nykma — CHANGES_REQUESTED** [review]
Good enough.

File: src/controller/graphql/resolve.rs (hunk 1 above)
Comment (ZhongFuze) [reply src/controller/graphql/resolve.rs]: Done

**Review by nykma — APPROVED** [review]
Accepted.

**Pull request merged**